src/includes/docs-details.html

```diff
@@ -10,7 +10,7 @@ <h2 id="docs">Description</h2>
 <div id="top-doc">
 	<!--Populated by JS-->
 </div>
-<h2 id="field-list">Field List</h2>
+<h2 id="field-list" style="display: none;">Field List</h2>
```

Comment:
Let's avoid inline styles -- put it in the .css instead.

Reply:
lol turns out there was already a `#field-list-header { display: none; }` but this was using `id="field-list"` :man_facepalming: